pkg/interfaces/contracts/standalone-utils/IEulerToken.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.7.0;
+
+import "../solidity-utils/openzeppelin/IERC20.sol";
+
+interface IEulerToken is IERC20 {
+    /**
+     * @dev Convert an eToken balance to an underlying amount, taking into account current exchange rate
+     * @param balance eToken balance, in internal book-keeping units (18 decimals)
+     * @return Amount in underlying units, (same decimals as underlying token)
+     * https://github.com/euler-xyz/euler-contracts/blob/master/contracts/modules/EToken.sol#L104
```

Comment:
Are these contracts final?
Maybe consider referencing a given commit in master rather than master itself in case the file is updated?

Reply:
added references